src/form.js

```diff
@@ -852,6 +852,8 @@ define([
                     this.moveMug(conflict, "rename", oldId);
                 }
             }
+
+            return updates;
```

Comment:
If this is going to be returned, I think the docstring for `handleMugRename` should be updated to describe the shape of this object.

Another thought I'm having is that there may be an event that gets fired when a mug is renamed. If that were the case then it might not be necessary to return this.